### ISSUE TITLE
key_masks

### ISSUE THREAD

**Issue by changzhou-pku** [kang205/SASRec#2]
"key_masks = tf.sign(tf.abs(tf.reduce_sum(keys, axis=-1))) # (N, T_k)" at line 185 in modules.py indicates that the key_masks depend on the embedding of those keys, but why not use the original "key_masks = tf.sequence_mask(keys_length, tf.shape(keys)[1])   # (N, T_k)"?
